qt/scientific_interfaces/Indirect/test/ConvFunctionModelTest.h

```diff
@@ -0,0 +1,131 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+//     NScD Oak Ridge National Laboratory, European Spallation Source
+//     & Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+#ifndef MANTIDQT_CONVFUNCTIONMODELTEST_H_
+#define MANTIDQT_CONVFUNCTIONMODELTEST_H_
+
+#include <cxxtest/TestSuite.h>
+
+#include "IndirectFunctionBrowser/ConvFunctionModel.h"
+#include "MantidAPI/FunctionFactory.h"
+
+using namespace Mantid::API;
+using namespace MantidQt::CustomInterfaces::IDA;
+
+class ConvFunctionModelTest : public CxxTest::TestSuite {
+public:
+  //   /// WorkflowAlgorithms do not appear in the FrameworkManager without this
+  //   line
+  ConvFunctionModelTest() {}
+
+  static ConvFunctionModelTest *createSuite() {
```

Comment:
Ditto about not having `createSuite` and `destroySuite` when you have no constructor (or one that does nothing)